torchci/clickhouse_queries/commit_jobs_query/query.sql

```diff
@@ -1,158 +1,164 @@
-    workflow.artifacts_url AS github_artifact_url,
-    if(
-      workflow.conclusion = ''
-      and workflow.status = 'queued',
-      'failure',
-      workflow.conclusion
-    ) as conclusion,
-    workflow.html_url,
-    '' AS log_url,
-    DATE_DIFF(
-      'SECOND', workflow.created_at, workflow.run_started_at
-    ) AS queue_time_s,
-    0 AS duration_s,
-    '' as line,
-    [] as captures,
-    0 as line_num,
-    [] as context,
-    '' AS runner_name,
-    workflow.head_commit.author.email AS authorEmail
-  FROM
-    workflow_run workflow
-  WHERE
-    workflow.event != 'workflow_run' -- Filter out workflow_run-triggered jobs, which have nothing to do with the SHA
-    AND workflow.event != 'repository_dispatch' -- Filter out repository_dispatch-triggered jobs, which have nothing to do with the SHA
-    AND workflow.head_sha = {sha : String}
-    AND workflow.repository.full_name = {repo : String}
+    SELECT
+        job.started_at AS time,
+        workflow.head_sha AS sha,
+        job.name AS job_name,
+        workflow.name AS workflow_name,
+        job.id,
+        workflow.id AS workflow_id,
+        workflow.artifacts_url AS github_artifact_url,
+        job.conclusion,
+        job.html_url,
+        IF(
+            {repo: String } = 'pytorch/pytorch',
+            CONCAT(
+                'https://ossci-raw-job-status.s3.amazonaws.com/log/',
+                job.id:: String
+            ),
+            CONCAT(
+                'https://ossci-raw-job-status.s3.amazonaws.com/log/',
+                {repo: String },
+                '/',
+                job.id:: String
+            )
+        ) AS log_url,
+        if(
+            job.started_at = 0,
+            0,
+            DATE_DIFF('SECOND', job.created_at, job.started_at)
+        ) AS queue_time_s,
+        if(
+            job.completed_at = 0,
+            0,
+            DATE_DIFF('SECOND', job.started_at, job.completed_at)
+        ) AS duration_s,
+        job.torchci_classification.line as line,
+        job.torchci_classification.captures as captures,
+        job.torchci_classification.line_num as line_num,
+        job.torchci_classification.context as context,
+        job.runner_name AS runner_name,
+        workflow.head_commit. 'author'.'email' AS authorEmail
+    FROM
+        workflow_job job final
+        INNER JOIN workflow_run workflow final ON workflow.id = job.run_id
+    WHERE
+        job.name != 'ciflow_should_run'
+        AND job.name != 'generate-test-matrix'
+        AND workflow.event != 'workflow_run' -- Filter out workflow_run-triggered jobs, which have nothing to do with the SHA
+        AND workflow.event != 'repository_dispatch' -- Filter out repository_dispatch-triggered jobs, which have nothing to do with the SHA
+        AND workflow.head_sha = {sha: String }
+        AND job.head_sha = {sha: String }
+        AND workflow.repository. 'full_name' = {repo: String } --         UNION
+        --         -- Handle CircleCI
```

Comment:
Could we get rid of circle CI here too?

Reply:
Removed, I'm still not sure if I should ingest the circleci tables